### PR TITLE
Added oxli to pep257 maketarget

### PR DESCRIPTION
**Pull request by SensibleSalmon**
For the `clean` make target, should we also be removing all `.pyc` files in scripts? 'cause we purge tests and khmer, but not scripts.
- [x] Is it mergeable?
- [x] Did it pass the tests?
- [x] If it introduces new functionality in scripts/ is it tested?
  Check for code coverage with `make clean diff-cover`
- [x] Is it well formatted? Look at `make pep8`, `make diff_pylint_report`,
  `make cppcheck`, and `make doc` output. Use `make format` and manual
  fixing as needed.
- [x] Did it change the command-line interface? Only additions are allowed
  without a major version increment. Changing file formats also requires a
  major version number increment.
- [x] Is it documented in the ChangeLog?
  http://en.wikipedia.org/wiki/Changelog#Format
- [x] Was a spellchecker run on the source code and documentation after
  changes were made?
- [x] Is the Copyright year up to date?

### CODE REVIEW

**Comment by ctb** [comment]
On Mon, Aug 03, 2015 at 10:54:14AM -0700, Jake Fenton wrote:
> For the `clean` make target, should we also be removing all `.pyc` files in scripts? 'cause we purge tests and khmer, but not scripts.

Not necessary but probably ok to do it anyway.

**Comment by bocajnotnef** [comment]
@ctb @mr-c @luizirber Ready for merge

File: Makefile [hunk 1]
```diff
@@ -73,7 +73,7 @@ clean: FORCE
 	cd lib && ${MAKE} clean || true
 	cd tests && rm -rf khmertest_* || true
 	rm -f $(EXTENSION_MODULE)
-	rm -f khmer/*.pyc lib/*.pyc
+	rm -f khmer/*.pyc lib/*.pyc scripts/*.pyc
```
Comment (mr-c) [inline Makefile]: oxli/*.pyc tests/*.pyc

**Comment by mr-c** [comment]
Otherwise this LGTM

**Comment by bocajnotnef** [comment]
@mr-c Updated; Merge?

**Comment by mr-c** [comment]
LGTM; make mergable and self-merge

**Pull request merged**